src/frontends/tensorflow/src/op/conv_3d_backprop.cpp

```diff
@@ -14,91 +14,101 @@ namespace tensorflow {
 namespace op {
 
 OutputVector translate_conv_3d_backprop_input_v2_op(const NodeContext& node) {
-    auto ng_filter = node.get_input(1);
-    auto ng_out_backprop = node.get_input(2);
+    TENSORFLOW_OP_VALIDATION(node, node.get_input_size() >= 2, "Conv3DBackpropInput must have at least three inputs.");
```

Comment:
at least three inputs, but why >= 2?
should be > 2 or >= 3?

Reply:
Great catch! Fixed.